proxy/http/HttpSM.cc

```diff
@@ -549,8 +549,9 @@ HttpSM::attach_client_session(HttpClientSession * client_vc, IOBufferReader * bu
 
   NetVConnection* netvc = client_vc->get_netvc();
 
-  ats_ip_copy(&t_state.client_info.addr, netvc->get_remote_addr());
-  t_state.client_info.port = netvc->get_local_port();
+  ats_ip_copy(&t_state.client_info.src_addr, netvc->get_remote_addr());
+  ats_ip_copy(&t_state.client_info.dst_addr, netvc->get_local_addr());
+  t_state.client_info.dst_addr.port() = netvc->get_local_port();
```

Comment:
client_info is the only one where we initialize both sides of the connection info right now because it's the only one where the port member had a different meaning before--should we init both sides for server_info? where would be best to do that/does it make sense?